### PR TITLE
JMeter 5.0

### PR DESCRIPTION
**Pull request by double16**
JMeter 5.0 with support for including OpenJFX libraries when not using the Oracle JDK.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -64,7 +72,7 @@ dependencies {
         exclude group: 'commons-pool2', module: 'commons-pool2'
         exclude group: 'xml-apis', module: 'xml-apis'
     }
-    compile("kg.apc:jmeter-plugins-webdriver:${jmPluginVersion}"){
+    compile("kg.apc:jmeter-plugins-webdriver:2.3"){
```
Comment (foragerr) [inline build.gradle:75]: @double16 why are you going to a fixed webdriver version here?

Comment (foragerr) [reply build.gradle:75]: Maybe this line needs an update instead?
https://github.com/jmeter-gradle-plugin/jmeter-gradle-plugin/blob/787fc6a9a60fbc99d7daf3a05cb867ce0da0fef2/build.gradle#L19

File: src/main/groovy/net/foragerr/jmeter/gradle/plugins/TaskJMInit.groovy [hunk 2]
```diff
@@ -95,7 +99,7 @@ class TaskJMInit extends DefaultTask {
                 writer.flush();
                 writer.close();
             } catch (IOException ioe) {
-                throw new GradleException("Couldn't create temporary property file " + f.getName() + " in directory " + workDir.getPath(), ioe);
+                throw new GradleException("Couldn't create temporary property file " + f.getName() + " in directory " + project.jmeter.workDir.getPath(), ioe);
```
Comment (foragerr) [inline src/main/groovy/net/foragerr/jmeter/gradle/plugins/TaskJMInit.groovy]: You seem to've generally replaced `workDir` with `binDir`, why is this particular line back to `workDir`?

**Review by foragerr — CHANGES_REQUESTED** [review]

File: build.gradle (hunk 1 above)
Comment (double16) [reply build.gradle:75]: This particular plugin has departed from the common version number of the other plugins. The others have a latest version of '1.4.0'. The `jmeter-plugins-webdriver` is actually `3.0` now. Do you think it's better to stay at `1.4.0` for this one too and let the user of the plugin change the version number?

Comment (foragerr) [reply build.gradle:75]: In which case, what you did makes sense 👍

**Review by foragerr — APPROVED** [review]

**Comment by foragerr** [comment]
Thank you for the contribution!

**Comment by wkurniawan07** [comment]
Hi @foragerr, may I know if a release that includes this PR will be published soon?

**Pull request merged**